examples/showrtf.c

```diff
@@ -99,7 +99,7 @@ static void * SDLCALL CreateFont(const char *name, RTF_FontFamily family, int ch
             TTF_style |= TTF_STYLE_ITALIC;
         if ( style & RTF_FontUnderline )
             TTF_style |= TTF_STYLE_UNDERLINE;
-        TTF_SetFontStyle(font, style);
+        TTF_SetFontStyle(font, TTF_style);
```

Comment:
This should go into SDL2 branch too (and even to SDL-1.2 branch, as far as I can see..)

Reply:
c4492f95f472d25cdc7783267692664c1f618cbc